### PR TITLE
added missing german translations

### PR DESCRIPTION
**Pull request by dabenzel**
Added missing german translations and fixed one mistake

### CODE REVIEW

**Comment by danharrin** [comment]
Thanks

File: packages/forms/resources/lang/de/components.php [hunk 1]
```diff
@@ -52,6 +56,22 @@
 
     ],
 
+    'checkbox_list' => [
+
+        'buttons' => [
+
+            'deselect_all' => [
+                'label' => 'Alle deselektieren',
```
Comment (dissto) [inline packages/forms/resources/lang/de/components.php:64]: @dabenzel @zepfietje 

How about "Auswahl aufheben". 
I personally never heard someone say "deselektieren" in my life 😊

File: packages/forms/resources/lang/de/components.php [hunk 2]
```diff
@@ -52,6 +56,22 @@
...
+            ],
+
+            'select_all' => [
+                'label' => 'Alle selektieren',
```
Comment (dissto) [inline packages/forms/resources/lang/de/components.php:68]: Same here "Alle/s auswählen" 🤔

File: packages/forms/resources/lang/de/components.php (hunk 1 above)
Comment (dabenzel) [reply packages/forms/resources/lang/de/components.php:64]: Good idea, you can issue a PR for that if you want

File: packages/forms/resources/lang/de/components.php (hunk 2 above)
Comment (dabenzel) [reply packages/forms/resources/lang/de/components.php:68]: s.o.

File: packages/forms/resources/lang/de/components.php (hunk 1 above)
Comment (zepfietje) [reply packages/forms/resources/lang/de/components.php:64]: Should `select_all` also be updated then?

**Pull request merged**